python/mujinwebstackclient/controllerwebclientraw.py

```diff
@@ -280,3 +337,89 @@ def CallGraphAPI(self, query, variables=None, headers=None, timeout=5.0):
             raise ControllerGraphClientException(_('Unexpected server response %d: %s') % (statusCode, raw), statusCode=statusCode, response=response)
 
         return content['data']
+
+    def _DefaultCallback(self, response: Dict[str, Any]):
+        # tmp: simply print the response
+        print(response)
+
+    async def _OpenWebSocketConnection(self):
+        self._websocket = await websockets.connect(
+            uri='ws%s' % self._graphEndpoint[len('http'):],
```

Comment:
Can we construct `graphEndpoint` here using `self._baseurl` instead of getting it from self?